code/modules/antagonists/heretic/items/heretic_blades.dm

```diff
@@ -282,3 +292,18 @@
 		heretic_datum.try_draw_rune(user, target, drawing_time = 14 SECONDS) // Faster than pen, slower than cicatrix
 		return ITEM_INTERACT_BLOCKING
 	return NONE
+
+// Weaker blade variant given to people so they can participate in the heretic arena spell
+/obj/item/melee/sickly_blade/training
+	name = "\improper imperfect blade"
+	desc = "A blade given to those who cannot accept the truth, out of pity. \
+		May it act as a blessing in the short time it remains alongside you."
+	force = 17
+	armour_penetration = 0
+
+/obj/item/melee/sickly_blade/training/Initialize(mapload)
+	. = ..()
+	QDEL_IN(src, 60 SECONDS)
```

Comment:
This should probably be set in the arena effect that grants the blade, rather than being innate to the item.